sccp/figures/common.py

```diff
@@ -557,7 +556,7 @@ def flattenWeightedProjs(data, factors, projs):
 
     weightedProjs = projs @ factors[1]
 
-    weightedProjs = weightedProjs / np.max(np.abs(weightedProjs),axis=0)
+    #weightedProjs = weightedProjs / np.max(np.abs(weightedProjs),axis=0)
```

Comment:
@pascoesean Uncomment and I can approve! This is to make sure the biggest value is from -1 or 1